src/boilerplates/boilerplates.ts

```diff
@@ -66,51 +67,78 @@ export function makeCommandBoilerplates(program: Command) {
         .command("create-kit-embedded-wallet-nextjs-starter")
         .description("Clone a starter boilerplate for Sequence Embedded Wallet integrated with Sequence Kit and Next.js")
         .option(
-        "--waas-config-key <waas_key>",
-        "WaaS config key for this project"
+            "--waas-config-key <waas_key>",
+            "WaaS config key for this project"
         )
         .option(
-        "--project-access-key <access_key>",
-        "Project access key for Sequence requests"
+            "--project-access-key <access_key>",
+            "Project access key for Sequence requests"
         )
         .option(
-        "--google-client-id <google_client_id>",
-        "Google client ID to be used during authentication"
+            "--google-client-id <google_client_id>",
+            "Google client ID to be used during authentication"
         )
         .option(
-        "--apple-client-id <apple_client_id>",
-        "Apple client ID to be used during authentication"
+            "--apple-client-id <apple_client_id>",
+            "Apple client ID to be used during authentication"
         )
         .option(
             "--wallet-connect-id <wallet_connect_id>",
             "Wallet Connect ID to be used during authentication"
         )
         .option(
-        "--verbose",
-        "Show additional information in the output"
+            "--verbose",
+            "Show additional information in the output"
         )
         .action((options) => {
             createEmbeddedWalletNextjs(program, options);
         });
 
+    comm
+        .command("create-embedded-wallet-linking-starter")
+        .description("Clone a starter boilerplate for Sequence Embedded wallet linking demo integrated with React")
+        .option(
+            "--waas-config-key <waas_key>",
+            "WaaS config key for this project"
+        )
+        .option(
+            "--project-access-key <access_key>",
+            "Project access key for Sequence requests"
+        )
+        .option(
+            "--google-client-id <google_client_id>",
+            "Google client ID to be used during authentication"
+        )
+        .option(
+            "--wallet-connect-id <wallet_connect_id>",
+            "Wallet Connect ID to be used during authentication"
+        )
+        .option(
+            "--verbose",
+            "Show additional information in the output"
+        )
+        .action((options) => {
+            createWalletLinkingEmbeddedWallet(program, options);
+        })
+        
```

Comment:
Change this identation to:
 .option(
 "--verbose",
 "Show additional information in the output"
)